CMakeLists.txt

```diff
@@ -102,6 +102,12 @@ foreach(_backend ${IREE_ALL_HAL_DRIVERS})
   set(IREE_HAL_DRIVER_${uppercase_backend} OFF CACHE BOOL "" FORCE)
 endforeach()
 
+# For cross compilation towords Android, we don't want LLVM JIT HAL driver.
+if(ANDROID)
+  message(STATUS "Removed LLVM JIT HAL driver for Android build")
+  list(REMOVE_ITEM IREE_HAL_DRIVERS_TO_BUILD LLVM)
+endif()
```

Comment:
Should this block go above line 97 instead? 

Also for the logging, could restructure to (pseudocode)

```
if(ANDROID and LLVM in IREE_HAL_DRIVERS_TO_BUILD)
  // remove from lsit
)
```

Reply:
Good point. Fixed.